src/test/js/shared/custom_steps.js

```diff
@@ -14,8 +14,8 @@ module.exports = function() {
         this.fillField('password', '111');
         this.scrollPageToBottom();
         this.click('Sign in');
-        this.waitInUrl('/authorize', 60);
-        this.seeInField('#state', 'attemptone');
+        this.waitInUrl('/login', 60);
+        this.retry({retries:3, minTimeout:1000}).seeInField('#state', 'attemptone');
```

Comment:
not being able to wait for authorize makes this line the thing that handles the wait - so it'll pretty much always fail the first attempt. After the change will the url be /login with all the parameters? If so we can look for the state being present in the url rather than being hidden on the page.

Reply:
Yep, the state is in the URL. Have adjusted to get it from here.